multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -157,6 +200,13 @@ write_files:
           "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services"
       done
 
+  - path: /etc/rkthack
```

Comment:
Actually, how about `rkt-chroot-escape-wrapper`? I think its non-obvious to most people that the purpose of this command is to escape the chroot rather then enter the mount namespace (we are already in the host mount namespace).